nntrainer/include/model_loader.h

```diff
@@ -80,6 +80,10 @@ class ModelLoader {
    * @param[in/out] layer layer to be loaded
    * @param[in] layer_name name of the layer to be loaded
    */
+  int loadLayerConfigIniCommon(dictionary *ini, std::shared_ptr<Layer> &layer,
```

Comment:
Need to fix the doxygen and could you let me know what is the diff common and without common?

Reply:
done.